index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portofolio page</title>
+    <link rel="stylesheet" href="style.css"> 
+</head>
+<body>
+    <header class="container">
+        <nav class="nav">
+        <a class="my-logo" href="index.html">
+        <img src="assets/images/My%20Logo.svg" alt="Portofolio logo"> 
+        </a>
+        <a class="hamburger" href="#">
+        <img src="assets/images/burger.svg" alt="Hambugrer_menu">   
+        </a>
+    </nav>
+    </header>
+    <section class="main">
+      <h1 class="header">I'am Sentayhu <br>  Glad to see <br>you!</h1>
+      <p class="intro">
+          I’m a software developer! I can help you build a product , 
+          feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+<a class="lets-connect" href="#">LETS CONNECT
+</a>
+<br>
+<div class="connect-via">
```

Comment:
- [ ] Please make sure you're following the template's margin and padding. At the moment, you have two elements overlapping each other, kindly fix this issue and add a bit more margin to the elements to better match the template.

![](https://i.imgur.com/Uji0UHs.png)